CMakeLists.txt

```diff
@@ -136,13 +136,13 @@ ELSE()
         SET(CMAKE_HIP_ARCHITECTURES ${Kokkos_HIP_ARCHITECTURES})
       ENDIF()
     ENDIF()
-    IF(${Kokkos_VERSION} VERSION_GREATER_EQUAL "4.3.01")
+    IF(${Kokkos_VERSION} VERSION_GREATER_EQUAL "4.4.01")
       MESSAGE(STATUS "Found Kokkos version ${Kokkos_VERSION} at ${Kokkos_DIR}")
-      IF((${Kokkos_VERSION} VERSION_GREATER "4.4.99"))
+      IF((${Kokkos_VERSION} VERSION_GREATER "4.5.99"))
         MESSAGE(WARNING "Configuring with Kokkos ${Kokkos_VERSION} which is newer than the expected develop branch - version check may need update")
       ENDIF()
     ELSE()
-      MESSAGE(FATAL_ERROR "Kokkos Kernels ${KokkosKernels_VERSION} requires Kokkos 4.3.01 or greater (found ${Kokkos_VERSION})")
+      MESSAGE(FATAL_ERROR "Kokkos Kernels ${KokkosKernels_VERSION} requires Kokkos 4.4.01 or greater (found ${Kokkos_VERSION})")
```

Comment:
Same as above, the new minimum should be 4.4.00

Reply:
We used 4.4.01 for the release and merge to master though, shouldn't we stay consistent?

Reply:
Yes that's correct